configure

```diff
@@ -388,6 +388,7 @@ msg "inspecting environment"
 
 CFG_OSTYPE=$(uname -s)
 CFG_CPUTYPE=$(uname -m)
+ENDIAN=$(printf '\1' | od -dAn)
```

Comment:
In order to minimize the possible impact of this change, could this move into just the mips block below?